Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -36,7 +36,7 @@ public enum Compression { None, Much, Lots, NoRotation }; // easily understandab
         public bool clientAuthority;
 
         // is this a local player with authority over his own transform?
-        bool isLocalPlayerWithAuthority => isLocalPlayer && clientAuthority;
+        bool isLocalPlayerWithAuthority => hasAuthority && clientAuthority;
```

Comment:
nice fix but it's not obvious why we use hasAuthority over isLocalPlayer. please explain in a comment as usual :)

Reply:
@vis2k Review latest please.